### ISSUE TITLE
All objects always available, even if rejected or unapproved...

### ISSUE THREAD

**Issue by ursomniac** [dominno/django-moderation#133]
I'm trying to do a parallel test from the shell to what's in the docs.

> > > from myapp.models import Event
> > > ee = Event(title='foo', date='2015-06-01', start_time='12:00:00')
> > > ee.save()
> > > ee.pk
> > > 17L

As I understand it, this should be unapproved and the follow should NOT happen.

> > > z = Event.objects.get(pk=17)
> > > 
> > > z
> > > <Event: 17: 2015-06-01 | foo>

According to the docs, this SHOULD return:
Traceback (most recent call last):
DoesNotExist: YourModel matching query does not exist.

Also, rejected items likewise also return (e.g., a rejected item has pk=9)

> > > z = Event.objects.get(pk=9)
> > > 
> > > z
> > > <Event: 9: 2015-06-01 | REJECTED!>

Thoughts?  Suggestions?

**Comment by ryuusenshi** [comment]
Are you sure you've created ``moderator.py``, registered the model and ran ``auto_discover()`` in your project level ``urls.py``?

**Comment by ursomniac** [comment]
Yup.   I'm looking at the code in the repo here and it doesn't match my local copy.   I suspect that I'm a rev behind (I'm using Django 1.7 which is likely relevant since the newest rev deals with 1.7 support).

**Comment by ursomniac** [comment]
Nope - I've installed the latest rev, and I still have the problem.  Unapproved/rejected objects still return from a Event.objects.get() call.

**Comment by ursomniac** [comment]
Furthermore, going through the admin causes an exemption.  "Registration Error at /admin/event/event/add/"   <class 'testlowell.apps.event.models.Event'> has been registered with Moderation.

I suspect that the rev I downloaded isn't stable?  Or not for Django 1.7.8?

**Comment by ursomniac** [comment]
I've backed out to Django 1.6.  It ALMOST works, but it appears that the problem (the original one I posted) is that visible_until_rejected is not being honored.   I have this moderator class defined:

class EventModerator(GenericModerator):
    notify_user = False
    notify_moderator = False
    auto_approve_for_superusers = True
    auto_approve_for_staff = False
    visible_until_rejected = False

But an Event that is created by a user who is staff only shows up in the Event table.  It DOES also appear in the moderation_moderated table correctly ('pending').   In the User table, the user has "event | event | Can add event" and "event | event | Can change event" set --- is that an error on my part (and if so, what's the correct set of user permissions since if I don't have them set, I get the "You don't have permission to edit anything" error).
